src/nccl_ofi_api.c

```diff
@@ -294,8 +294,20 @@ ncclResult_t nccl_net_ofi_regMr_v7(void *comm, void *data, int size, int type,
 ncclResult_t nccl_net_ofi_regMr(void *comm, void *data, size_t size, int type,
 				void **mhandle)
 {
-	int ret = 0;
+	return nccl_net_ofi_regMrDmaBuf(comm,
+									data,
```

Comment:
I think something is fairly wrong with the indenting here.

Reply:
I use a tabwidth of 4. I have proposed a clang format file and you rejected it.

Reply:
The project has a tabwidth of 8.